src/app.module.ts

```diff
@@ -10,14 +10,18 @@ import { HashtagsModule } from './hashtags/hashtags.module';
 import { AuthModule } from './auth/auth.module';
 import { PasswordEntity } from './auth/passwords.entity';
 import { SessionsEntity } from './auth/sessions.entity';
+import { ConfigModule } from '@nestjs/config';
 
+ 
 @Module({
   imports: [
+    ConfigModule.forRoot(),
     TypeOrmModule.forRoot({
+      host: process.env.DB_HOST,
       type: 'postgres',
-      username: 'mooadmin',
-      password: 'moopass',
-      database: 'moodb',
+      username: process.env.DB_USERNAME,
```

Comment:
for the non-docker version, the .env file is not read automatically. can use something like `node-env` to make that happen. 

as such this project will not run without docker right now if I merge this.

Reply:
Hi @championswimmer  , for non-docker also it is going to take the .env file, which i have already tested , the nestjs/config uses the .env file internally, which loads the file and update the process.env object

I will commit the .env file with default value. so, if anyone pull the project and run the application which can run without any problem.
thanks.

Reply:
Sound good!